tests/data/output/annotations/basic/output-uischema.json

```diff
@@ -16,9 +16,6 @@
     "justwarning": {
         "ui:warning": "Warning!"
     },
-    "nonreadonly": {
```

Comment:
Same as above - do we want to have a different pass-through logic for the default values ?

Reply:
Yes, if it's a `false` or not set, we shouldn't emit it. It defaults to `false` in the JSON Schema. IOW, emit as smallest JSON Schema as possible. Is it ok?

Reply:
Yeah, I think it's okay - the reason I had it is basically it could be maybe confusing for the person writing the jellyschema - `where did my keyword go` but otherwise is fine - I think this is as with the other comment - maybe let's just let people use it and we will see what happens :)

Reply:
I understand, but don't forget one thing - JSON Schema is like a build artifact - no one should care about it. People should care about JellySchema vs UI form only. This JSON Schema is here (playground), just to show them what's on going. But not tell them that they should check it. Maybe we can hide later, when everything will be working, dunno.